docs/How-to-use-kaito-in-aks-arc.md

```diff
@@ -0,0 +1,137 @@
+
+## Prerequisite
+1.	Before you begin, please make sure you have the following details from your infrastructure administrator:
+    - An AKS cluster that's up and running.
+    - We recommend using Linux machine for this feature.
+    - Your local kubectl environment configured to point to your AKS cluster.
+        - Run `az aksarc get-credentials --resource-group <ResourceGroupName> --name <ClusterName>  --admin` to download the kubeconfig file.
+2.	Make sure your HCI cluster is enabled with GPU, you can ask your infrastructure administrator to set it up for you. You also need to identify the right VM SKUs for your AKS cluster before creating the node pool. The instruction can be found at [use GPU for compute-intensive workloads](https://learn.microsoft.com/en-us/azure/aks/hybrid/deploy-gpu-node-pool).
+3.	Make sure the helm and kubectl are installed in your local machine.
+    - If you need to install or upgrade, please see instruction from [Install Helm](https://helm.sh/docs/intro/install/).
+    - If you need to install kubectl, please see instructions from [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+## Create a GPU Node Pool
+<details>
+<summary><b>Using Azure Portal</b></summary>
+<div align="middle">
+  <img src="img/aksarc_nodepool_creation_portal.png" width=80% title="create nodepool from azure portal" alt="create nodepool from azure portal">
+</div>
+</details>
+
+Run following Az command to provision node pool, available GPU sku can be found [here](https://learn.microsoft.com/en-us/azure/aks/hybrid/deploy-gpu-node-pool#supported-vm-sizes)
+
+```bash
+az aksarc nodepool add --name "samplenodepool" --cluster-name "samplecluster" --resource-group "sample-rg" –node-vm-size "samplenodepoolsize" –os-type "Linux"
+```
+
+### Validation
+1.	After node pool creation command succeeds, you can confirm whether the GPU node is provisioned using `kubectl get nodes`.
+```
+NAME              STATUS   ROLES           AGE   VERSION
+moc-l06l9ruvcd6   Ready    <none>          58d   v1.29.4
+moc-l9f0lh9ro95   Ready    control-plane   58d   v1.29.4
+moc-le4aoguwyd9   Ready    <none>          49d   v1.29.4
+```
+2.	Please also ensure the node has allocatable GPU cores using command 
+```bash
+kubectl get node moc-l1i9uh0ksne -o yaml | grep -A 10 "allocatable:"
+```
+```
+  allocatable:
+    cpu: 31580m
+    ephemeral-storage: "95026644016"
+    hugepages-1Gi: "0"
+    hugepages-2Mi: "0"
+    memory: 121761176Ki
+    nvidia.com/gpu: "2"
+    pods: "110"
+  capacity:
+    cpu: "32"
+    ephemeral-storage: 103110508Ki
+```
+
+## Deploy KAITO via Helm
+1.	Git clone the KAITO repo to your local machine
+2.	Install KAITO operator using command 
+```bash
+helm install workspace ./charts/kaito/workspace --namespace kaito-workspace --create-namespace
+```

```

Comment:
You'd better add a note here: User does not need to install the gpu-provisioner as the node will be provisioned manually.